config/rbac/kustomization.yaml

```diff
@@ -10,6 +10,9 @@ resources:
 - bindablekinds_viewer_rolebinding.yaml
 - servicebinding_controller_role.yaml
 - servicebinding_controller_rolebinding.yaml
+# operators supproted out of the box
+- opstree_redis_clusterrole.yaml
+#- opstree_redis_viewer_rolebinding.yaml
```

Comment:
This line is commented. If not required, remove this and the corresponding YAML file.